cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst

```diff
@@ -0,0 +1,24 @@
+PIM Updates and Migrations
+==========================
+
+As the Serenity offer is designed with simplicity in mind, you don't have to do anything to
+benefit from the latest patches and releases. The Akeneo team will take care of the upgrades
+as quickly as possible.
+
+You will get the latest stable (minus alpha, beta or pre-released) version available.
```

Comment:
'minus' for  me minus sound like the symbol, perhaps 'not'  could be without ambiguity

Reply:
Maybe, just removing it as it sounds normal that alpha, beta, pre-released are not stable releases.